Release 0.8.0

korneliakobiela opened the pull request:
4e219d5 Remove necessery registerElements
99792f5 Fix new page wizard element add Component class
a8a8cdd Fix New Page wizard Child Components
66d73a9 Add the Feature usage for Radio button (#470)
0069f19 Fix ruler element

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

lslachciak commented on closet-component-packages/components/image/package.json:
```diff
@@ -8,29 +8,6 @@
   "template": "<img class=\"ui-picture\" src=\"#\" alt=\"\" style=\"height: 100px; width: 100px; display:block; background:#cacaca\">",
   "resizable": true,
   "draggable": true,
-  "data": [
```
do we need this change in release?
we only update package-lock.

lmslachciak reviewed (commented):
I tested this in WATT with links:
http://localhost:3000/demos?path=1.1%2Fexamples%2Fwearable%2FUIComponents%2Fcontents%2Fcontrols%2Fdimmer.html
http://localhost:3000/demos?path=1.1%2Fexamples%2Fmobile%2FUIComponents%2Fcomponents%2Fgallery%2Fgallery.html

In both cases in Chromium v. 80 I'm getting errors after going to live preview in WATT
**page-wizard-element.js:82 Uncaught TypeError: Cannot read property 'templateFileName' of undefined**
    at page-wizard-element.js:82
    at Array.forEach (<anonymous>)
    at HTMLElement.value (page-wizard-element.js:77)
    at HTMLElement.value (page-wizard-element.js:56)
    at new t (page-wizard-element.js:33)
    at HTMLElement.value (toolbar-element.js:199)
    at HTMLElement.value (toolbar-element.js:27)
    at HTMLElement.value (dress-element.js:142)
    at e.value (panel-manager.js:127)
    at e.value (panel-manager.js:159)
(anonymous) @ page-wizard-element.js:82
value @ page-wizard-element.js:77
value @ page-wizard-element.js:56
t @ page-wizard-element.js:33
value @ toolbar-element.js:199
value @ toolbar-element.js:27
value @ dress-element.js:142
value @ panel-manager.js:127
value @ panel-manager.js:159
value @ stage-manager.js:48
(anonymous) @ brackets-editor.js:105
(anonymous) @ main.js:18
Promise.then (async)
(anonymous) @ main.js:16
Promise.then (async)
(anonymous) @ main.js:14
Promise.then (async)
consumeCloset @ main.js:12
(anonymous) @ brackets-editor.js:103
(anonymous) @ main.js:20
Promise.then (async)
(anonymous) @ main.js:18
Promise.then (async)
consumeCloset @ main.js:16
(anonymous) @ brackets-editor.js:102
(anonymous) @ main.js:18
Promise.then (async)
consumeCloset @ main.js:16
value @ brackets-editor.js:101
(anonymous) @ brackets-editor.js:174
i @ bootstrap:19
(anonymous) @ bootstrap:83
(anonymous) @ bootstrap:83
**toolbar-element.js:289 Uncaught (in promise) TypeError: Cannot read property 'EDITOR' of undefined**
    at HTMLElement.value (toolbar-element.js:289)
    at e.value (stage-manager.js:397)
    at brackets-editor.js:105
    at main.js:18

Commit:
Release 0.8.0

7ec6f67 Fix page wizard element error
4e219d5 Remove necessery registerElements
99792f5 Fix new page wizard element add Component class
a8a8cdd Fix New Page wizard Child Components
66d73a9 Add the Feature usage for Radio button (#470)
0069f19 Fix ruler element

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

korneliakobiela commented:
All elements were cleared. It works. CI will not work - we don't have a config for Custom Elements and JSDom doesn't support it by default.

lslachciak commented:
Both urls work now

lmslachciak reviewed (changes requested):
@korneliakobiela could you disable npm run coverage-test and create new issue for enabling it
so we will be sure that other parts of CI are passing
